cli/cmd/options.go

```diff
@@ -189,6 +203,14 @@ func makeInstallUpgradeFlags(defaults *l5dcharts.Values) ([]flag.Flag, *pflag.Fl
 	return flags, installUpgradeFlags, nil
 }
 
+func initResourceMaps(values ...map[string]interface{}) {
+	for _, value := range values {
+		if _, ok := value["Resources"]; !ok {
```

Comment:
I think this should be?

```suggestion
		if _, ok := value["resources"]; !ok {
```

When I look at the output of this, Prometheus has the following field `Resources: {}`; I assume it's empty because it's supposed to be the `resources: ..` field.

Reply:
Yeah there's something odd going on there, but I think this is the minimum change we require to fix the bug for now.